CHANGES.md

```diff
@@ -1,5 +1,25 @@
 # Changes
 
+## edge-22.10.1
+
+This edge release fixes some sections of the Viz dashboard that were appearing
+blank, and adds an optional PodMonitor resource to the Helm chart which allows
+having an easier integration with the Prometheus Operator. It also includes many
```

Comment:
grammar nit: "allows having an easier integration" feels kind of awkward to me, maybe consider something like:
```suggestion
blank, and adds an optional PodMonitor resource to the Helm chart to enable
easier integration with the Prometheus Operator. It also includes many
```